backend/app/tests/api/routes/test_login.py

```diff
@@ -79,6 +82,13 @@ def test_reset_password(
     assert r.status_code == 200
     assert r.json() == {"message": "Password updated successfully"}
 
+    # * db asserts
```

Comment:
```suggestion
```

We can probably remove these comments, I guess they were there during development of these tests.

Reply:
Done